applications/jupyter/workloads.tfvars

```diff
@@ -28,24 +28,25 @@ cluster_membership_id = "" # required for private cluster, defaults to `cluster_
 
 ## JupyterHub variables
 namespace                         = "ml"
-gcs_bucket                        = "gcs-bucket-dsfhfh"
 create_gcs_bucket                 = true
-workload_identity_service_account = "jupyter-service-account"
+gcs_bucket                        = "gcs-bucket-dsfhfh"
```

Comment:
Can we change this to "gcs-bucket-<unique suffix"> and add a comment saying it needs to be unique across all projects?

Reply:
okay